buildpacks/nodejs-engine/src/main.rs

```diff
@@ -83,27 +90,34 @@ impl Buildpack for NodeJsEngineBuildpack {
             Requirement::parse(LTS_VERSION).expect("The default Node.js version should be valid")
         };
 
-        let target_release =
-            inv.resolve(&version_range)
-                .ok_or(NodeJsEngineBuildpackError::UnknownVersionError(
-                    version_range.to_string(),
-                ))?;
+        let target_release = inventory.resolve(&version_range).ok_or(
+            NodeJsEngineBuildpackError::UnknownVersionError(version_range.to_string()),
+        )?;
 
         log_info(format!(
             "Resolved Node.js version: {}",
             target_release.version
         ));
 
         log_header("Installing Node.js distribution");
-        context.handle_layer(
+        let dist_layer = context.handle_layer(
             layer_name!("dist"),
             DistLayer {
                 release: target_release.clone(),
             },
         )?;
 
+        let env = dist_layer.env.apply(Scope::Build, &Env::from_current());
+        let node_version = node::get_node_version(&env)
```

Comment:
Don't we already know the Node.js version we installed (I think it's in the `target_release` object)? It seems like this might be an extra command we don't need.